sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosFullTextPolicy.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Constants;
+import com.azure.cosmos.implementation.JsonSerializable;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * Full Text Search Policy
+ */
+public class CosmosFullTextPolicy {
+    @JsonProperty(Constants.Properties.DEFAULT_LANGUAGE)
+    private String defaultLanguage;
+    @JsonProperty(Constants.Properties.PATHS)
+    private List<CosmosFullTextPath> paths;
+    private JsonSerializable jsonSerializable;
+
+    /**
+     * Constructor
+     */
+    public CosmosFullTextPolicy() { this.jsonSerializable = new JsonSerializable(); }
+
+    /**
+     * Gets the default language for cosmosFullText.
+     *
+     * @return the default language for cosmosFullText.
+     */
+    public String getDefaultLanguage() {
+        return defaultLanguage;
+    }
+
+    /**
+     * Sets the default language for cosmosFullText.
+     * @param defaultLanguage the default language for cosmosFullText.
+     * @return CosmosFullTextPolicy
+     */
+    public CosmosFullTextPolicy setDefaultLanguage(String defaultLanguage) {
+        this.defaultLanguage = defaultLanguage;
```

Comment:
Doesn't this need to set it in the `jsonSerializable`?

Reply:
It seems It's a pure POJO not wrapping jsonserializable like others but using JsonAnnotations.
Would be a good learning point on when to use what.

Reply:
In the Java SDK we are using jsonSerializable in the top POJO, and using JsonAnnotations for the inner POJO's. Just following what is been followed across the SDK.